src/inference/dev_api/cpp_interfaces/interface/ie_iinfer_request_internal.hpp

```diff
@@ -253,7 +299,9 @@ class INFERENCE_ENGINE_API_CLASS(IInferRequestInternal) : public std::enable_sha
     std::vector<std::shared_ptr<const ov::Node>> _parameters;  //!< A vector of function inputs
     std::vector<std::shared_ptr<const ov::Node>> _results;     //!< A vector of function outputs
     std::map<std::string, PreProcessDataPtr> _preProcData;     //!< A map of pre-process data per input
-    int m_curBatch = -1;                                       //!< Current batch value used in dynamic batching
+    //!< A map of user passed blobs via ov::runtime::InferRequest::set_tensors/set_input_tensors for network inputs
```

Comment:
Try not to use public API. Even in coments.

Reply:
agree